### DISCUSSION TITLE
Allow to create an anchor or link from headings

### DISCUSSION THREAD

**Discussion by jmvtrinidad** [community/community#124090] [Discussions]
### Select Topic Area

Product Feedback

### Body

It will be good to share a link that will be able to anchor to a certain heading/header from the contents of any post or comments inside the discussion.
I prefer to use the discussion instead of wiki to allow comments on a certain guide/discussion.

**Update**
Using [html works](https://github.com/orgs/community/discussions/124090#discussioncomment-9546611) to redirect user to a certain part, 
but it would be great if github has similar experience just like on the readme where I can just hover on some headings
![image](https://github.com/community/community/assets/7194300/ca50bb9b-ade6-4e19-bcad-7b32c38f1c15)

**Comment by zerobugszone** [comment]
To enable sharing links that anchor to a specific heading or header within a discussion post or comment, you can use HTML anchor tags and IDs. This approach allows users to jump directly to a particular section of a discussion, enhancing navigation and user experience. Here's how you can implement this:

    Add IDs to Headings:
    Ensure that each heading in your discussion posts has a unique ID attribute. This will serve as the anchor point.

    html

<h2 id="introduction">Introduction</h2>
<p>Welcome to the discussion...</p>

<h2 id="details">Details</h2>
<p>Here are the detailed information...</p>

<h2 id="conclusion">Conclusion</h2>
<p>This concludes the discussion...</p>

Create Anchor Links:
To link to these specific headings, you can create URLs that include the ID of the target heading.

html

<a href="#introduction">Go to Introduction</a>
<a href="#details">Go to Details</a>
<a href="#conclusion">Go to Conclusion</a>

Using Anchor Links in Comments:
Users can include these anchor links in their comments to reference specific sections of the discussion.

html

<p>Great point mentioned in the <a href="#details">Details section</a>.</p>

Ensuring Compatibility:
Make sure your discussion platform supports HTML anchor tags and IDs. Most modern web applications do, but it's good to confirm this with your platform's documentation or support team.

Example in Context:
Here’s an example of how it might look in a discussion:

Original Post:

html

<h2 id="introduction">Introduction</h2>
<p>Welcome to the discussion on web development best practices...</p>

<h2 id="details">Details</h2>
<p>Here are the detailed steps and methodologies...</p>

<h2 id="conclusion">Conclusion</h2>
<p>This concludes the discussion with a summary...</p>

User Comment:

html

    <p>I found the information in the <a href="#details">Details section</a> very useful for my project.</p>

By implementing these steps, you can enhance the interactivity and usability of your discussions, making it easier for users to reference and navigate specific parts of the content.

**Reply by jmvtrinidad** [reply]
Thank you for this, this works
But Im looking for the behaviour similar to readme for ease of use
![image](https://github.com/community/community/assets/7194300/72e54e3a-8b97-42d8-b419-be1db7544c79)
